Updates to writing assistance section (#2198)
MicrosoftDocs/edge-developer: microsoft-edge/privacy-whitepaper/index.md

> 

Reviewer: Suggest changing the two h3's in this article to h4's to make page structure perceptible and clear:
Change:
```
### Microsoft News
```
to:
```
#### Microsoft News
```

Change:
```
### Office 365
```
to:
```
#### Office 365
```

The feature is only available for users with a non-child Microsoft account.  The feature is not available for users signed into Microsoft Edge with a work or school account.

> <!-- ====================================================================== -->
> ## Personalization

Reviewer: ```suggestion

<!-- ====================================================================== -->
## Personalization

```
per repo conventions for readability/ease of maintenance and for well-formed robust Markdown.

You may modify your interests or opt-out of personalized ads from the [Ad settings](https://account.microsoft.com/privacy/ad-settings) on the [Microsoft privacy dashboard](https://account.microsoft.com/privacy/).

> > [!NOTE]

Reviewer: The tagging is probably broken since Note tagging line is gone.  Such Alerts (eg Note colored rectangle) are deprecated.  Alternatives are:
```
**Note: ** Foo.
```

```
#### Make a label for the note as an h4 heading
```
DCG:   https://learn.microsoft.com/en-us/contribute/markdown-reference#alerts-note-tip-important-caution-warning - "Avoid Notes, Tips, and Important boxes. Readers tend to skip over them. It's better to put that info directly into the article text.

If you need to use alerts, limit them to one or two per article. Multiple notes should never be next to each other in an article."

Proposed fix that avoids deprecated Markdown tagging:  The simplest quickest fix is delete the " >" from the "Opting" paragraph.  A more ambitious / structural fix is:

```
#### Ad settings

You can modify your interests or opt-out of personalized ads from the [Ad settings](https://account.microsoft.com/privacy/ad-settings) on the [Microsoft privacy dashboard](https://account.microsoft.com/privacy/).

Opting out of personalized ads on the [Microsoft privacy dashboard](https://account.microsoft.com/privacy/) does not turn off the collection and use of your browsing history for personalization of search results and content in your news feed. You can turn off the collection and use of your Microsoft Edge browsing history for personalized search results and news. Go to `edge://settings/privacy`. In **Personalize your web experience** turn off the **Improve your web experience by allowing Microsoft to use your browsing history from the account for personalizing advertising, search, news and other Microsoft services** setting. If you stop sharing the data, Microsoft no longer collects and uses your browsing history to personalize ads, search results, and news. For more information about personalization in Microsoft Edge, see [Microsoft Edge browsing history for personalized advertising and experiences](https://support.microsoft.com/help/4532583).
```
changes made above: 
* invented an h4 heading
* changed "may" to "can".
* Removed "> " from "Opting" paragraph.

You may modify your interests or opt-out of personalized ads from the [Ad settings](https://account.microsoft.com/privacy/ad-settings) on the [Microsoft privacy dashboard](https://account.microsoft.com/privacy/).

> > Opting out of personalized ads on the [Microsoft privacy dashboard](https://account.microsoft.com/privacy/) does not turn off the collection and use of your browsing history for personalization of search results and content in your news feed. You can turn off the collection and use of your Microsoft Edge browsing history for personalized search results and news. Go to `edge://settings/privacy`. In **Personalize your web experience** turn off the **Improve your web experience by allowing Microsoft to use your browsing history from the account for personalizing advertising, search, news and other Microsoft services** setting. If you stop sharing the data, Microsoft no longer collects and uses your browsing history to personalize ads, search results, and news. For more information about personalization in Microsoft Edge, see [Microsoft Edge browsing history for personalized advertising and experiences](https://support.microsoft.com/help/4532583).

Reviewer: The tagging is probably broken since Note tagging line is gone.  Such Alerts (eg Note colored rectangle) are deprecated.  Alternatives are:
```
**Note: ** Foo.
```

```
#### Make a label for the note as an h4 heading
```
DCG:   https://learn.microsoft.com/en-us/contribute/markdown-reference#alerts-note-tip-important-caution-warning - "Avoid Notes, Tips, and Important boxes. Readers tend to skip over them. It's better to put that info directly into the article text.

If you need to use alerts, limit them to one or two per article. Multiple notes should never be next to each other in an article."

To enable diagnostic data collection, go to `edge://settings/privacy`. Turn on the **Optional diagnostic data** setting. To turn off diagnostic data collection for Microsoft Edge on Windows 10 and later, open **Start** > **Settings** > **Privacy** and select **Diagnostics & feedback**. To turn off diagnostic data collection for all other platforms, go to `edge://settings/privacy`. Turn off the **Help improve Microsoft products by sending optional diagnostic data about how you use the browser, websites you visit, and crash reports** setting. The applications hosting Microsoft Edge WebView may collect other data that is governed by the data collection management of the developer and relevant privacy policies.

<!-- ====================================================================== -->
## Writing assistance

> Microsoft Edge provides enhanced spelling and grammar checking as you type in the browser. Go to `edge://settings/languages` and select **Microsoft Editor** under **Use writing assistance toggle**. If you select **Microsoft Editor** to provide enhanced spelling and grammar checking, Microsoft Edge will send your typed text and a service token to a Microsoft cloud service over a secure HTTPS connection. The service token doesn't contain any user-identifiable information. A Microsoft cloud service then processes the text to detect spelling and grammar errors in your text. All your typed text sent to Microsoft are deleted immediately after processing occurs. No data is stored for any period of time. If you select **Basic** under **Use writing assistance** toggle in `edge://settings/languages`, the service will perform only local spellchecking on the device and no data will be sent to the cloud for spellchecking services. 

Reviewer: ```suggestion
Microsoft Edge provides enhanced spelling and grammar checking as you type in the browser.  To use enhanced spelling and grammar checking, go to `edge://settings/languages` and select **Microsoft Editor** under the **Use writing assistance** toggle. If you select **Microsoft Editor**, Microsoft Edge sends your typed text and a service token to a Microsoft cloud service over a secure HTTPS connection. The service token doesn't contain any user-identifiable information. A Microsoft cloud service then processes the text to detect spelling and grammar errors in your text. All your typed text sent to Microsoft is deleted immediately after processing occurs. No data is stored for any period of time. If you select **Basic** under **Use writing assistance** toggle in `edge://settings/languages`, the service will perform only local spellchecking on the device and no data will be sent to the cloud for spellchecking services. 
```
generally use present tense, but for settings, keeping future tense might be best.

To turn off all spelling and grammar capabilities, toggle **Use writing assistance toggle** off. 
 

> When you add a new language to Microsoft Edge, the browser downloads the dictionary for the new language to the device using HTTPS. The dictionary is used for the basic spellcheck service or for languages not supported by Editor spelling and grammar checking. Deleting the language from your Microsoft Edge settings deletes the dictionary from the device.To add or remove words in your local dictionary, go to `edge://settings/languages` and under **Customize dictionary**, select **Add or delete words**.

Reviewer: ```suggestion
When you add a new language to Microsoft Edge, the browser downloads the dictionary for the new language to the device by using HTTPS. The dictionary is used for the basic spellcheck service or for languages that aren't supported by Editor spelling and grammar checking. Deleting the language from your Microsoft Edge settings deletes the dictionary from the device.  

To add or remove words in your local dictionary, go to `edge://settings/languages`, and then under **Customize dictionary**, select **Add or delete words**.
```

 Custom dictionary is not available in **Guest** mode.

> 

Reviewer: fyi feel free to add h4 headings, skipping h3's, to give more skimmable points of entry and break up long sections - like labelling paragraphs to differentiate their main focus.

<!-- ====================================================================== -->
## Writing assistance

Microsoft Edge provides enhanced spelling and grammar checking as you type in the browser.  To use enhanced spelling and grammar checking, go to `edge://settings/languages` and select **Microsoft Editor** under the **Use writing assistance** toggle. 

If you select **Microsoft Editor**, Microsoft Edge sends your typed text and a service token to a Microsoft cloud service over a secure HTTPS connection. The service token doesn't contain any user-identifiable information. A Microsoft cloud service then processes the text to detect spelling and grammar errors in your text. All your typed text that's sent to Microsoft is deleted immediately after processing occurs. No data is stored for any period of time. 

> If you select **Basic** under **Use writing assistance** toggle in `edge://settings/languages`, the service will perform only local spellchecking on the device, and no data will be sent to the cloud for spellchecking services. 

Reviewer: ```suggestion
If you select **Basic** under the **Use writing assistance** toggle in `edge://settings/languages`, the service will perform only local spellchecking on the device, and no data will be sent to the cloud for spellchecking services. 
```

> To turn off all spelling and grammar capabilities, toggle **Use writing assistance toggle** off. 

Reviewer: ```suggestion
To turn off all spelling and grammar capabilities, turn off the **Use writing assistance toggle** toggle.
```
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/t/toggle

 
When you add a new language to Microsoft Edge, the browser downloads the dictionary for the new language to the device over HTTPS. The dictionary is used for the basic spellcheck service or for languages that aren't supported by Microsoft Editor spelling and grammar checking. Deleting the language from your Microsoft Edge settings deletes the dictionary from the device.

> To add or remove words in your local dictionary, go to `edge://settings/languages`, and then under **Customize dictionary**, select **Add or delete words**.

Reviewer: ```suggestion
To add or remove words in your local dictionary, go to `edge://settings/languages`, and then under **Customize dictionary**, click the **Add word** button, or click the **X** button next to a word that you added, to remove it.
```

>  Custom dictionary is not available in **Guest** mode.

Reviewer: ```suggestion
The custom dictionary is not available in **Guest** mode.
```
